include/drivers/flash.h

```diff
@@ -36,12 +36,20 @@ struct flash_pages_layout {
 };
 #endif /* CONFIG_FLASH_PAGE_LAYOUT */
 
+/**
+ * Flash memory parameters
+ */
+struct flash_parameters {
+	uint8_t erase_value; /* Pattern representing erased flash */
```

Comment:
Or "Byte value of erased flash".  Not terribly important, but "pattern" to me suggests a mask rather than a value.